Project Euler/problem_01/sol1.py

```diff
@@ -0,0 +1,24 @@
+"""
+Problem Statement:
+If we list all the natural numbers below 10 that are multiples of 3 or 5,
+we get 3,5,6 and 9. The sum of these multiples is 23.
+Find the sum of all the multiples of 3 or 5 below N.
+"""
+"""
```

Comment:
Not a huge edit but I see you have been following a consistent style, could you remove the double `"""`